onnxruntime/core/providers/dml/OperatorAuthorHelper/MLOperatorAuthorPrivate.h

```diff
@@ -57,6 +57,30 @@ IMLOperatorKernelCreationContextPrivate : public IMLOperatorKernelCreationContex
     ) const noexcept PURE;
 };
 
+interface __declspec(uuid("1d2e1226-a918-4236-8775-175cf1f52c9a"))
+IMLOperatorKernelCreationContextPrivate1 : public IMLOperatorKernelCreationContextPrivate 
```

Comment:
Do we actually need a stable ABI for private interfaces? Seems like this is a relict causing unnecessary overhead.